src/components/accordion/_test_examples.js

```diff
@@ -0,0 +1,15 @@
+export { EXAMPLE_ACCORDION };
+
+const EXAMPLE_ACCORDION = {
```

Comment:
```suggestion
export const EXAMPLE_ACCORDION = {
```

Reply:
this is neater for one case but if things are added in the future then wouldn't we want the original set up? that's how it's set up in other files i believe

Reply:
we can move to this - but it won't be consistent with some of the others already merged, i preferred it like this initially because it is the same format as the import statement - making it easier to ensure you are exporting and importing the exact same thing